lib/docker-sync/docker_compose_session.rb

```diff
@@ -26,11 +26,24 @@ def down
 
     private
 
+    def docker_compose_binary_exists?
```

Comment:
we should call this the legacy binary

Reply:
Thx for the suggestion!
Made the change in [19e0ec1](https://github.com/EugenMayer/docker-sync/pull/838/commits/19e0ec1d1dfeb6caed13f516a1ce65e47fa8482e)